.github/test-image.yaml

```
@@ -0,0 +1,35 @@
+name: Testing
+
+on:
+  pull_request:
```

ow about triggering it on a push to the main branch as well? :thinking: 
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```